### PR TITLE
fix: Charge Stations icons have incorrect color

### LINKED ISSUE #512: Charge stations icons have the incorrect colour when zoomed out
Screenshot from 2021-05-28 10-12-39

When zoomed out the dots are black but they should be same turquoise as the icons.

Screenshot from 2021-05-28 10-13-54

### PR DESCRIPTION
**Pull request by jpidelatorre**
Icons for charge stations appeared black when zoomed out.
Changed the color for the icon.

Closes #512

### CODE REVIEW

File: app/component/map/tile-layer/ChargingStations.js [hunk 1]
```diff
@@ -61,7 +61,15 @@ class ChargingStations {
...
     if (this.tile.coords.z <= this.config.chargingStations.smallIconZoom) {
-      return drawStopIcon(this.tile, geom, 'charging-station');
+      return drawStopIcon(
+        this.tile,
+        geom,
+        'charging-station',
+        null,
+        null,
+        null,
+        { 'mode-charging-station': '#00b096' },
```
Comment (andreashelms) [inline app/component/map/tile-layer/ChargingStations.js]: Could you define color in `config.hbnext.js` line 92 instead of adding it here?

Comment (jpidelatorre) [reply app/component/map/tile-layer/ChargingStations.js]: That would be a good idea, but we should do that for all the icons. I could start with this one.

Comment (andreashelms) [reply app/component/map/tile-layer/ChargingStations.js]: I already did that for bus, car and rail:

```
colors: {
        primary: '#9fc727',
        iconColors: {
            'mode-bus': '#ff0000',
            'mode-car': '#007AC9',
            'mode-rail': '#008000',
        },
    },
```

Comment (jpidelatorre) [reply app/component/map/tile-layer/ChargingStations.js]: I will add the rest of the colors, then.

**Pull request merged**